### PR TITLE
Tutorials: Clean up plando guide a bit; explain datapackage page. Add…

### PR DESCRIPTION
**Pull request by alwaysintreble**
… link to the weighted settings page in advanced tutorial.

### CODE REVIEW

File: WebHostLib/static/assets/tutorial/archipelago/plando_en.md [hunk 1]
```diff
@@ -66,38 +64,9 @@ list of specific locations both in their own game or in another player's game.
...
-* [Rogue Legacy](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/rogue-legacy/Names/LocationName.py)
-* [Slay the Spire](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/spire/Locations.py)
-* [Subnautica](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/subnautica/locations.json)
-* [Super Metroid](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/sm/variaRandomizer/graph/location.py#L132)
-* [Timespinner](https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/timespinner/Locations.py#L13)
-
-A list of all available items and locations can also be found in the [server's datapackage](/api/datapackage).
+### Available Items and Locations
+
+A list of all available items and locations can be found in the [website's datapackage](/api/datapackage). The items and locations will be in the `"item_name_to_id"` and `"location_name_to_id"` sections of the relevant game. You do not need the quotes but the name must be entered in the same as it appears on that page and is caps-sensitive.
```
Comment (Berserker66) [inline WebHostLib/static/assets/tutorial/archipelago/plando_en.md:69]: for regular users you probably want to skip the `/api`

Comment (JarnoWesthof) [reply WebHostLib/static/assets/tutorial/archipelago/plando_en.md:69]: Personally i find exposing the /api/datapackage scary, as it only represents the latest released version of AP on archipelago.gg, but that might not be what version your AP a server you connecr to runs (incase of running games outside of archipelago.gg) so no game should ever use it as its source of information

Comment (alwaysintreble) [reply WebHostLib/static/assets/tutorial/archipelago/plando_en.md:69]: ah actually didn't mean to make the `/api` section there. will fix that. @JarnoWesthof what do you mean exactly? kind of the purpose of a lot of these tutorials is intended for current stable releases. If something isn't working how someone is expecting and the issue is using an old version then I would chalk that up to user error.

**Comment by Ijwu** [comment]
Thanks!

**Pull request merged**